README.md

```diff
@@ -1,7 +1,81 @@
-# Secure forward proxy plugin for the Caddy web server
+# Secure forward proxy for the Caddy web server
+
+This package registers the `http.handlers.forward_proxy` module, which acts as an HTTPS proxy for accessing remote networks.
+
+## :warning: Experimental!
+
+This module is EXPERIMENTAL. We need more users to test this module for bugs and weaknesses before we recommend its use from within surveilled networks or regions with active censorship. Do not rely on this code in situations where personal safety, freedom, or privacy are at risk.
+
+**You can help by:**
+
+- Safely deploying this module
+- Trying to break it
+- Contributing to the code and tests in this repo to make it better
+
+We are also seeking experienced maintainers who have experience with these kinds of technologies and who are interested in continuing its development.
+
+**Expect breaking changes.**
+
+## Features
+
+- HTTP/1.1 and HTTP/2 support
```

Comment:
Should HTTP/3 be mentioned here?

Reply:
If it works :man_shrugging: :sweat_smile: I haven't tested that.

Reply:
At least HTTP/3 is the reason tests are failing, cause of that header. Would disabling http/3 for forwardproxy be an option? 🤔

Reply:
H3 proxy works at least in Chrome. It's a real use case.

Reply:
Then it might be good to see if the tests can be updated to support HTTP/3, since it works with Chrome. And yeah, let's add it to the README. :100: